apps/web/src/app/api/auth/google/callback/route.ts

```diff
@@ -48,14 +48,23 @@ export async function GET(request: NextRequest): Promise<Response> {
     const name = claimsParser.getString('name')
     const email = claimsParser.getString('email')
 
+    // Read invitation token from cookie
+    const invitationToken = cookiesStore.get('latitude_invitation_token')?.value
```

Comment:
Why the outflow is involved?

As I see this 

1. Admin creates an invitation link `/invitations/:token`
2. Invitee receives it and if it's valid this creates the membership and add cookie session. 

After those two steps inviee is now part of the workspace.